bootstrap.py

```diff
@@ -287,40 +336,61 @@ def gen_email():
         try:
             with open(q_path) as file:
                 questions = yaml.load(file.read())
-            print('found quiz')
+            less_counter.quiz_outcome = 'found'
         except Exception as err:
             questions = False
-            print(err)
+            less_counter.quiz_outcome = 'none'
         if questions:
             quiz = models.Quiz(title=meta['title'], users=DEFAULT_USER, description=meta['description'])
             quiz_id = await quiz.update_or_create('title')
             meta['quiz'] = quiz_id
             question_order = 1
             for _, val in questions.items():
-                qustion = models.Question(**val)
-                qid = await qustion.update_or_create(*val.keys())
-                qq = models.QuizQuestions(quiz=quiz_id, question=qid, question_order=question_order)
-                question_order += 1
-                await qq.update_or_create('question', 'quiz')
-                print('question created')
-        lesson = models.Lesson(**meta)
-        lid = await lesson.update_or_create(*meta.keys())
+                try:
+                    qustion = models.Question(**val)
+                    qid = await qustion.update_or_create(*val.keys())
+                    qq = models.QuizQuestions(quiz=quiz_id, question=qid, question_order=question_order)
+                    question_order += 1
+                    await qq.update_or_create('question', 'quiz')
+                    less_counter.quiz_details_done += 1
+                except Exception as err:
+                    print(err)
```

Comment:
jeśli nie chcesz stack trace'a wypisywać, to polecam przynajmniej repr(err) wyświetlić, samo err nawet nie wypisze jaki to typ błędu